nodejs/week1/avg.js

```diff
@@ -0,0 +1,20 @@
+"use strict"; // allows us to use only declared variables
+let n = 0;
+let m = 0;
+for (let i = 0; i < process.argv.length; i++) {
+  // in case that process.argv array shows us
+  // in 0.index Node executable path and in 1.index
+  // script path - we need to sort it out with checking
+  // if it is Number or not. Also it helps us to sort out
+  // string values
+  if (!isNaN(parseInt(process.argv[i]))) {
+    n += parseInt(process.argv[i]);
+    m += 1;
+  }
+}
+// if no arguments are provided
+if (isNaN(n / m)) {
+  console.log("Please enter some numbers");
```

Comment:
Function still executes and calculates average number even when I enter a string as an argument.

Reply:
No, it works :) It showing the error message if user provided only strings or an empty string.
If there are in input both number and strings - it shows average of numbers. And of course average of numbers if user will provide only numbers :)

Reply:
That's what I’m saying: if I enter both strings and numbers function still executes. But it probably should gives en error to provide all valid numbers, don't you think?😊